cdap-docs/admin-manual/source/security/authorization.rst

```diff
@@ -57,58 +58,57 @@ individual extensions for configuring properties specific to that extension:
 :ref:`Security extension properties <appendix-cdap-default-security>`, which are specified
 in ``cdap-site.xml``, begin with the prefix ``security.authorization.extension.config``.
```

Comment:
Here after the sentry thing put a a link to ranger 

After this section can you add a section for Ranger and just say for ranger refer this and link to https://github.com/caskdata/cdap-security-extn/wiki/CDAP-Ranger-Extension